data/hails.txt

```diff
@@ -2802,6 +2880,11 @@ phrase "hostile bounty hunter"
 		"They ponied up"
 		"I was promised"
 		"I was given"
+		"A little bird gave me"
+		"I was hired for"
+		"My boss has paid me"
+		"My contractor paid me"
+		"You've got someone who has paid me"
```

Comment:
This one doesn't really work.

Reply:
"You've got someone who has paid me a large sum of money to kill you."
"I was hired for a fat purse to turn you into space debris."

I disagree, they sound alright to me.

Reply:
"I was hired for a fat purse..." is metaphorical.

"You've got someone who has paid me" is uncultured schoolyard slang. That might fly in pirate hails, but the bounty hunters aren't supposed to sound like dopes.

Reply:
kk removed cause you're picky. will commit it later.